site-src/geps/pending-geps.md

```diff
@@ -0,0 +1,58 @@
+#### Gateway API Enhancment Proposal Status November 2023
```

Comment:
I think this is a useful idea, and I'm really not sure how else to track this, but I'm hesitant to add it in this location (our official docs). Maybe it belongs in `hack/` for lack of a better place? Just something to show that this is a temporary/working doc.

Reply:
I'll move to hack/